draft-ietf-lamps-x509-slhdsa.md

```diff
@@ -152,80 +148,80 @@ The fields in AlgorithmIdentifier have the following meanings:
 
-   id-alg-slh-dsa-shake-128f OBJECT IDENTIFIER ::= { sigAlgs TBD }
+   id-slh-dsa-shake-128f OBJECT IDENTIFIER ::= { sigAlgs 27 }
 
-   id-alg-slh-dsa-shake-192s OBJECT IDENTIFIER ::= { sigAlgs TBD }
+   id-slh-dsa-shake-192s OBJECT IDENTIFIER ::= { sigAlgs 28 }
 
-   id-alg-slh-dsa-shake-192f OBJECT IDENTIFIER ::= { sigAlgs TBD }
+   id-slh-dsa-shake-192f OBJECT IDENTIFIER ::= { sigAlgs 29 }
 
-   id-alg-slh-dsa-shake-256s OBJECT IDENTIFIER ::= { sigAlgs TBD }
+   id-slh-dsa-shake-256s OBJECT IDENTIFIER ::= { sigAlgs 30 }
 
-   id-alg-slh-dsa-shake-256f OBJECT IDENTIFIER ::= { sigAlgs TBD }
+   id-slh-dsa-shake-256f OBJECT IDENTIFIER ::= { sigAlgs 31 }
 ~~~
 
 The contents of the parameters component for each algorithm are absent.
 
-# SLH-DSA Signatures in PKIX
+# SLH-DSA Signatures
 
 SLH-DSA is a digital signature scheme built upon hash functions. The security of SLH-DSA relies on the presumed difficulty of finding preimages for hash functions as well as several related properties of the same hash functions.
 
-Signatures are used in a number of different ASN.1 structures.  As shown in the ASN.1 representation from {{!RFC5280}} below, in an X.509 certificate, a signature is encoded with an algorithm identifier in the signatureAlgorithm attribute and a signatureValue attribute that contains the actual signature.
+Signatures can be placed in a number of different ASN.1 structures.
+The top level structure for a certificate is given below as being
+illustrative of how signatures are frequently encoded with an
+algorithm identifier and a location for the signature.
 
 ~~~
-    Certificate  ::=  SEQUENCE  {
-        tbsCertificate       TBSCertificate,
-        signatureAlgorithm   AlgorithmIdentifier,
-        signatureValue       BIT STRING  }
+   Certificate  ::=  SEQUENCE  {
+      tbsCertificate       TBSCertificate,
+      signatureAlgorithm   AlgorithmIdentifier,
+      signatureValue       BIT STRING }
 ~~~
 
-Signatures are also used in the CRL list ASN.1 representation from {{!RFC5280}} below.  In a X.509 CRL, a signature is encoded with an algorithm identifier in the signatureAlgorithm attribute and a signatureValue attribute that contains the actual signature.
-
-~~~
-    CertificateList  ::=  SEQUENCE  {
-        tbsCertificate       TBSCertList,
-        signatureAlgorithm   AlgorithmIdentifier,
-        signatureValue       BIT STRING  }
-~~~
-
-The identifiers defined in {{sec-alg-ids}} can be used as the AlgorithmIdentifier in the signatureAlgorithm field in the sequence Certificate/CertificateList and the signature field in the sequence TBSCertificate/TBSCertList in certificates CRLs, respectively, {{!RFC5280}}.  The parameters of these signature algorithms are absent, as explained in {{sec-alg-ids}}.
-
-The signatureValue field contains the corresponding SLH-DSA signature computed upon the ASN.1 DER encoded tbsCertificate {{!RFC5280}}.
+The same algorithm identifiers in {{sec-alg-ids}} are used for signatures as are used
```

Comment:
Is there a doubling of "are used" in "are used for signatures as are used for public keys"?

Reply:
That is fine in English, I think it's the "in {{sec-alg-ids}}" that makes it awkward and isn't necessary so I'll remove that.